decidim-surveys/app/views/decidim/surveys/admin/surveys/_form.html.erb

```diff
@@ -17,6 +17,9 @@
       <div class="row column">
         <%= form.check_box :clean_after_publish, label: t("models.components.clean_after_publish", scope: "decidim.forms.admin"), aria: { label: :clean_after_publish } %>
       </div>
+      <div class="row column">
+        <%= form.check_box :allow_editing_answers, label: t("models.components.allow_editing_answers", scope: "decidim.forms.admin"), aria: { label: :allow_editing_answers } %>
```

Comment:
This should be below the "Allow answers" checkbox, and it should have a help text: `If active, answers from registered participants can be edited after submission.`

Reply:
fixed in f3c195ec15